hardware/Wireless-Products-Protocols-and-Applications.md

```diff
@@ -1,54 +1,61 @@
+- [3. Bluetooth products and applications](#3-bluetooth-products-and-applications)
+- [4. Z-Wave products and applications](#4-z-wave-products-and-applications)
 
-## Proprietary products and applications
+</details>
+
+# 1. Proprietary products and applications
 - Introduction: 
 
   Silicon Labs' proprietary wireless devices provide high-performance wireless connectivity and ultra-low power 32-bit microcontroller options. With support for major frequency bands in the 142 to 1050 MHz range and 2.4 GHz, these devices can be used to develop many wireless communication systems.
 - Chip name conventions:
 
   For proprietary products, there are many FG type chips which are classified with generations, packages, flash size, temperature grades, output power levels, and frequency bands. The general name convention likes:
-
-  ![](files/HW-Wireless-Products-Protocols-and-Applications/NameConvention-1.png)
-
-
+  <div align="center">
+  <img src="files\HW-Wireless-Products-Protocols-and-Applications\NameConvention-1.png">  
+  </div>
+  
   As you can see from the name convention, the proprietary chips include four series such as FG1,FG12,FG13,and FG14. The chips are classified with  P (Performance), B (Basic), and V (Value) performance grades. The chips are Tx/Rx type which include sub-G band, 2.4GHz band, and sub-G& 2.4GHz bands. The chips are classified with 2 temperature range as -40°-+85° and -40°-+125°. And there are many packages(QFN32, QFN48, QFN68, and BGA125) to select. 
 - Applications:
 
-  Meters(Gas, water, electricity), Door lockers, Electronic lables, Wireless toy, Wireless ranch, Remote controllers, etc.
-## Zigbee/Thread products and applications
+  Meters(Gas, water, electricity), Door locks, Electronic lables, Wireless toys, Wireless ranch, Remote controllers, etc.
+# 2. Zigbee/Thread products and applications
 - Introduction: 
 
-  Silicon Labs is the vendor of choice for OEMs developing Zigbee networking into their products. The Silicon Labs Zigbee platform is the most integrated, complete and feature rich Zigbee solution available — a family of Wireless SoCs, based on Arm® Cortex® processor and 2.4 GHz transceiver, together the most reliable, scalable and advanced Zigbee software and supported by best-in-class development tools.
+  Silicon Labs is the vendor of choice for OEMs developing Zigbee networking into their products. The Silicon Labs Zigbee platform is the most integrated, complete and feature rich Zigbee solution available — a family of Wireless SoCs, based on Arm® Cortex® processor and 2.4 GHz transceiver, together with the most reliable, scalable and advanced Zigbee software and supported by best-in-class development tools.
 - Chip name conventions:
 
   For Zigbee/Thread products, there are also many MG type chips which are classified with generations, packages, flash size, temperature grades, output power levels, and frequency bands. The general name convention likes:
```

Comment:
Change "general name convention likes:" to "general name convention is as follows:"

Reply:
Modified.